src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -231,11 +230,99 @@ private HttpRequestWrapper toHttpRequest(HookStageExecutionResult<EntrypointPayl
                 .build();
     }
 
+    private Future<BidRequest> parseBidRequest(HttpRequestWrapper httpRequest, List<String> errors) {
+        try {
+            final JsonNode bidRequestNode = bodyAsJsonNode(httpRequest.getBody());
+
+            final String referer = paramsExtractor.refererFrom(httpRequest);
+            ortbTypesResolver.normalizeBidRequest(bidRequestNode, errors, referer);
+
+            return Future.succeededFuture(jsonNodeAsBidRequest(bidRequestNode));
+        } catch (Exception e) {
+            return Future.failedFuture(e);
+        }
+    }
+
+    private Future<AuctionContext> extractAccountAndExecuteRawAuctionHooks(BidRequest bidRequest,
```

Comment:
Method name is a bit misleading - it does much more than just extracts account and executes hooks.

Reply:
renamed as executeRawAuctionHooksAndCreateAuctionContext